src/jquery.tagthis.js

```diff
@@ -218,13 +218,23 @@
 
         });
 
-        if (options && options.width && options.height){
-            wrapper = $('#tag-this--'+this[0].id);
-
-            $(wrapper).css({
-                'width' : options.width,
-                'height' : options.height
-            });
+        if (options.width || options.height || options.margin) {
+            wrapper = $('#tag-this--' + this[0].id);
+            if (options.width) {
+                $(wrapper).css({
+                    'width': options.width
+                });
+            }
+            if (options.height) {
+                $(wrapper).css({
+                    'height': options.height
+                });
+            }
+            if (options.margin) {
```

Comment:
I like the optional width & height, but I intentionally added only the ability to set those width & height programmatically because the tags inside the input are affected by those dimensions. The plugin itself won't benefit from being able to programmatically adjust margin, but margin can just as easily be set with CSS: `.tag-this { margin: 20px; }`